### PR TITLE
Copy clipboard tests

### PR DESCRIPTION
**Pull request by ben-d758**
**What**  
- add test for copy to clipboard feature on referral service information

**Why**  
- to test that the copy icon next to the referral service works correctly

**New packages installed**  
- installed Clipboardy package

### CODE REVIEW

File: cypress/integration/pages/home.spec.js [hunk 1]
```diff
@@ -105,4 +105,17 @@ context('Index page', () => {
...
+      cy.document().then( doc => {
+        doc.body.innerHTML = '<textarea data-testid="service-details">';
+      });
+      cy.get('[data-testid=service-details]').type('test{selectall}');
+      cy.document().then( doc => {
+        doc.execCommand('copy');
+      });
+      cy.task('getClipboard').should('contain', 'test');
+    });
+  });
```
Comment (kosiakkatrina) [inline cypress/integration/pages/home.spec.js:120]: Seems like we're not clicking the icon (?)
If I'm understanding correctly, this test gets the textarea directly and copies it. Maybe we can rewrite it to click on the icon instead?

Comment (elena-vi) [reply cypress/integration/pages/home.spec.js:120]: 1. click on icon
2. cy.task('getClipboard').should('contain', 'test');

File: components/Feature/VulnerabilitiesGrid/ResourceCard/index.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ const ResourceCard = ({
       <textarea id={`resource-${name}`} type="hidden" value={clipboardServiceDetails} hidden />
       <h3>
         {name}{' '}
-        <a title="copy" href="javascript:void(0)" onClick={() => copyToClipboard()}>
+        <a data-testid="copy-clipboard-icon" title="copy" href="javascript:void(0)" onClick={() => copyToClipboard()}>
```
Comment (elena-vi) [inline components/Feature/VulnerabilitiesGrid/ResourceCard/index.js:116]:
```suggestion
        <a id="copy-clipboard-icon" title="copy" href="javascript:void(0)" onClick={() => copyToClipboard()}>
```

File: cypress/integration/pages/home.spec.js [hunk 3]
```diff
@@ -101,4 +101,23 @@ context('Index page', () => {
...
   });
+
+  describe('When the copy icon next to a referral service is clicked', () => {
+    it('it copies the referral information to the clipboard', () => {
+      cy.get('[data-testid=accordion-item]')
+          .eq(0)
+          .click();
+      cy.get('[data-testid=resource-1]')
+          .eq(0)
+      cy.get('[data-testid=copy-clipboard-icon]').eq(0).click({ force: true });
```
Comment (elena-vi) [inline cypress/integration/pages/home.spec.js:112]:
```suggestion
      cy.get('#copy-clipboard-icon').eq(0).click({ force: true });
```

**Review by elena-vi — COMMENTED** [review]
instead of assigning special `data-testid attribites` to elements, can we just assign meaningful id's to things we care about

File: cypress/integration/pages/home.spec.js (hunk 3 above)
Comment (ben-d758) [reply cypress/integration/pages/home.spec.js]: Thanks @elena-vi . I've made those changes now

File: cypress/integration/pages/home.spec.js [hunk 4]
```diff
@@ -101,4 +101,23 @@ context('Index page', () => {
...
     });
   });
+
+  describe('When the copy icon next to a referral service is clicked', () => {
+    it('it copies the referral information to the clipboard', () => {
+      cy.get('[data-testid=accordion-item]')
+          .eq(0)
+          .click();
+      cy.get('[data-testid=resource-1]')
+          .eq(0)
```
Comment (Duslerke) [inline cypress/integration/pages/home.spec.js:111]: Is there an action missing?

**Review by Duslerke — APPROVED** [review]

**Pull request merged**